tests/Unit/PointwiseFunctions/Hydro/EquationsOfState/Python/Test_PolytropicFluid.py

```diff
@@ -1,19 +1,27 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
-import spectre.PointwiseFunctions.Hydro.EquationsOfState as spectre_eos
-
 import unittest
```

Comment:
Are the imports required to be alphabetized in `.py` files, or is that just with `#include` in `.cpp`/`.hpp` files?  If not, then everything looks good to me.

Reply:
We haven't established a rule for sorting imports yet. I started to use isort recently, which puts system modules before third-party modules, then alphabetizes.